docs/_static/custom.js

```diff
@@ -19,10 +19,24 @@ function openModal(modal) {
   modal.style.removeProperty('display');
 }
 
+function updateSetting(element) {
+  localStorage.setItem(element.name, element.checked);
+}
+
+function setFont(useSans) {
+  if (useSans === true) {
```

Comment:
I think you can remove the boolean comparison here?

```suggestion
  if (useSans) {
```

Reply:
Also, I think we should devise a more generalized system for applying classes to `<body>` according to a setting, as that'd probably be a common occurrence.